libs/lib-linker/src/lib/lib-linker.ts

```diff
@@ -0,0 +1,52 @@
+export type LinkerResult =
+  | 'OK'
+  | 'IDENTITY_UNKNOWN'
+  | 'IDENTITY_REGISTERED_ALREADY'
+  | 'IDENTITY_LINKED'
+  | 'IDENTITY_IDENTICAL'
+  | 'TOKEN_EXPIRED'
+  | 'UNKNOWN_ERROR'
+  | 'MESSAGE_SENT_TO_SUPPORT';
+
+export const LinkerResult = {
```

Comment:
I think with the power of TS you can cast all properties of object to given type automatically.